Osiris/Hacks/Misc.cpp

```diff
@@ -950,6 +950,28 @@ void Misc::preserveKillfeed(bool roundStart) noexcept
     }
 }
 
+void Misc::voteRevealer(GameEvent* event = nullptr) {
```

Comment:
`event` could be `GameEvent&` as the function does nothing when `event` is nullptr.